app/views/hub/verification_attempts/show.html.erb

```diff
@@ -57,8 +57,8 @@
                     <% end %>
                   </div>
                 </div>
-                <div class="grid-flex justify-flex-end">
-                  <div class="spacing-below-0"><%= link_to "#{timestamp(attempt.created_at)}", hub_verification_attempt_path(id:attempt.id), id: "time" %></div>
+                <div class="grid-flex column-column justify-flex-end">
```

Comment:
its a flex method - there's methods like column-column, row-column, column-row so that you can create different layouts for mobile and larger screens. since the default flex direction is row, row-row would be the implied default.

This change makes sure the two elements are stacked on top of one another instead of aligned side by side.